mail/mail-policy.go

```diff
@@ -73,114 +58,93 @@ func GetMailPolicy(
-	)
-	message = append(message, `Qualora tu abbia già provveduto, ignora questa comunicazione </p> `)
+	bodyData := BodyData{}
+	var tpl bytes.Buffer
+
+	cc := setBodyDataAndGetCC(policy, &bodyData)
+
+	templateFile := getTemplateByChannel(policy, "pay")
+
+	fillTemplate(templateFile, &bodyData, &tpl)
+
 	SendMail(
 		GetMailPolicy(
-			policy,
+			&policy,
 			"Paga la tua polizza"+" n° "+policy.CodeCompany,
 			true,
+			cc,
 			policy.PayUrl,
 			"Paga la tua polizza",
-			message,
+			tpl.String(),
 			false,
 			nil,
 		),
 	)
 }
 
 func SendMailSign(policy models.Policy) {
-	var message []string
-	message = append(message, `Puoi ora completare la sottoscrizione della tua polizza.`)
-	message = append(
-		message,
-		`<br>Clicca sul bottone sottostante per accedere alla procedura semplice e guidata di firma elettronica avanzata tramite 
-	utilizzo di un codice usa e getta che verrà inviato via sms sul tuo cellulare a noi comunicato.`,
-	)
-	message = append(
-		message,
-		`<br>Ti verrà richiesta l’adesione al servizio che è fornito in maniera gratuita da Wopta. Potrai prendere visione delle condizioni generali di servizio e delle caratteristiche tecniche.`,
-	)
-	message = append(
-		message,
-		`<br>Ultimata la procedura di firma potrai procedere al pagamento.<br>`,
-	)
-	message = append(message, `Qualora tu abbia già provveduto, ignora questa comunicazione </br> `)
+	bodyData := BodyData{}
+	var tpl bytes.Buffer
+
+	cc := setBodyDataAndGetCC(policy, &bodyData)
+
+	templateFile := getTemplateByChannel(policy, "sign")
+
+	fillTemplate(templateFile, &bodyData, &tpl)
+
 	SendMail(
 		GetMailPolicy(
-			policy,
+			&policy,
 			"Firma la tua polizza"+" n° "+policy.CodeCompany,
 			true,
+			cc,
 			policy.SignUrl,
 			"Firma la tua polizza",
-			message,
+			tpl.String(),
 			false,
 			nil,
 		),
 	)
 }
 
 func SendMailContract(policy models.Policy, at *[]Attachment) {
-	var message []string
-
-	message = append(
-		message,
-		`<p>ti confermiamo che la protezione offerta dalla tua polizza è ora attiva. 
-    in allegato trovi la documentazione firmata tramite l’utilizzo della Firma Elettronica. Salva e conserva i documenti con cura, ti serviranno in caso di sinistro.
-    Ti consigliamo di scaricare l’App di Wopta dagli store tramite il comodo QR code che trovi nell area sottostante per accedere 
-    alla tua area riservata nella quale troverai tutte le informazioni sulle polizze in tuo possesso e altri servizi a te riservati.</p>
-    Puoi usare anche questi canali per effettuare una denuncia di sinistro e verificare lo stato delle tue polizze e dei pagamenti.
-Seguici su nostri canali social o sul sito e scopri le iniziative a te riservate.
-    `,
-	)
+	bodyData := BodyData{}
+	var tpl bytes.Buffer
+
+	cc := setBodyDataAndGetCC(policy, &bodyData)
+
+	templateFile := getTemplateByChannel(policy, "sign")
```

Comment:
template is `sign` or should be another value?
In general create constants at the beginning of the file with these values